tests/test_middleware.py

```diff
@@ -133,6 +133,65 @@ def test_expired_token_forces_renewal(self, mock_middleware_random):
         self.assertEqual(expected_query, parse_qs(qs))
 
 
+@override_settings(OIDC_OP_AUTHORIZATION_ENDPOINT='http://example.com/authorize')
+@override_settings(OIDC_RP_CLIENT_ID='foo')
+@override_settings(OIDC_RENEW_TOKEN_EXPIRY_SECONDS=120)
+class RefreshOIDCAccessTokenMiddlewareTestCase(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.middleware = RefreshOIDCAccessToken()
```

Comment:
RefreshOIDCAccessToken() request an arg, should be something like:
  self.middleware = RefreshOIDCAccessToken(MagicMock)